README.md

```diff
@@ -130,17 +130,9 @@ html_theme_options = {
 }
 ```
 
-You can disable logos by setting `disable_ecosystem_logo` in `html_theme_options`:
-
-```python
-html_theme_options = {
-    "disable_ecosystem_logo": True,
-}
-```
-
 ## Tip: suggested site structure
```

Comment:
This will eventually be replaced by https://github.com/Qiskit/documentation/issues/1708